test/manual.test.ts

```diff
@@ -0,0 +1,86 @@
+import stytch = require("../lib");
+
+function env(name: string): string {
+  const val = process.env[name];
+  if (val) {
+    return val;
+  }
+  throw new Error(`Missing required environment variable: ${name}`);
+}
+
+function describeIf(name: string, cond: boolean, fn: () => void): void {
+  if (cond) {
+    describe(name, fn);
+  } else {
+    describe.skip(name, fn);
+  }
+}
+
+// These tests can't be run programmatically (even as integration tests), but it's still useful to
+// record their setup and expectations.
+describe("manual tests", () => {
+  let client: stytch.Client;
+
+  beforeEach(() => {
+    client = new stytch.Client({
+      project_id: env("PROJECT_ID"),
+      secret: env("SECRET"),
+      env: process.env["STYTCH_API_URL"] || stytch.envs.test,
+    });
+  });
+
+  // To run these tests, start a new session (it doesn't matter how) and paste its session_token
+  // and session_jwt values here.
+  //
+  // Do not commit any real tokens to this repo.
```

Comment:
Use `jose` to create a test JWKS and test JWTs to verify against inside the test itself. See [this file](https://github.com/stytchauth/web-backend/blob/3a0c3a2c07d436bf362e6b11556323dab20ee4fb/app/auth/utils/jwt.test.ts) for an example

Reply:
We should absolutely have non-manual tests for `authenticate_jwt_local` which doesn't need to make any network calls

Reply:
Thanks for the link! I had debated exposing `currentDate` to make this testable, but actually minting JWTs is an even better test.

Reply:
I ended up doing both. I think that exposing `currentDate` (as `current_date`) only on `authenticateJwtLocal` is the right balance of control and convenience. `authenticateJwt` (auto-fallback) always uses the current time, but an app may want to a frozen local time and fall back to a network call separately.